app/models/Wallet.scala

```diff
@@ -33,6 +34,8 @@ case class Wallet(pubKey: PublicKey) {
 
 object Wallet {
 
+  def fromId(id: String): Option[Wallet] = Base58.decode(id).filter(_.length == 32).map(id => Wallet(PublicKey @@ id)).toOption
```

Comment:
Still not sure whether its a right way to prevent creation of keys from arbitrary strings. 
Without filter its is possible to write `Wallet.fromId("true")` or even `Wallet.fromId("")`
@oskin1, please, have a more detail look at this problem.

Test should be added for this method.

Reply:
@jhnsmth use `Base16`